cocos/ui/UIEditBox/iOS/CCUIEditBoxIOS.mm

```diff
@@ -109,9 +109,8 @@ - (void)setTextInput:(UIView<UITextInput,CCUITextInput> *)textInput
 - (void)createSingleLineTextField
 {
     CCUISingleLineTextField *textField = [[[CCUISingleLineTextField alloc] initWithFrame:self.frameRect] autorelease];
-    textField.contentVerticalAlignment = UIControlContentVerticalAlignmentCenter;
```

Comment:
@anjandaffo Is there a reason to remove this default value? it goes with the discussion about defaults on android.